dependencyLibs/Core/src/org/droidplanner/core/mission/waypoints/DoLandStart.java

```diff
@@ -0,0 +1,55 @@
+package org.droidplanner.core.mission.waypoints;
+
+import com.MAVLink.common.msg_mission_item;
+import com.MAVLink.enums.MAV_CMD;
+
+import org.droidplanner.core.helpers.coordinates.Coord2D;
+import org.droidplanner.core.helpers.coordinates.Coord3D;
+import org.droidplanner.core.mission.Mission;
+import org.droidplanner.core.mission.MissionItem;
+import org.droidplanner.core.mission.MissionItemType;
+
+import java.util.List;
+
+public class DoLandStart extends SpatialCoordItem {
```

Comment:
The idea is that the classes in the `ClientLib` represent the interface for apps to use, while the classes in `Core` represent the mavlink specific implementation for that interface. I'm not updating old class files yet, but attempt to stick to that convention with newly created classes.

And by making it easier, I was referring to marshaling logic, like in `ProxyUtils` not having to specify the full path to differentiate the interface and implementation classes, and also for someone reading the source being able to identify what each class is for :) .